### PR TITLE
client updates

### PR DESCRIPTION
**Pull request by jasnames**

### CODE REVIEW

File: content/developer-guide/identity-dev/dev-guide-least-privilege.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 uid: LeastPrivilege
 ---
 
-# Least privileges
+# Least privilege
 
-When creating new users or client credential clients, configure them using the principle of Least Privilege. For more information, see the Cybersecurity & Infrastructure Security Agency article [Least Privilege](https://us-cert.cisa.gov/bsi/articles/knowledge/principles/least-privilege). Assign users and clients the lowest privileged role necessary to access resources.
+When creating new users or client-credentials clients, configure them using the principle of least privilege, which is that a process, user, or program should be able to access only the information and resources that are necessary for its legitimate purpose. Assign users and clients the lowest privileged role necessary to access resources.
+
+Ensure that clients which are not in use are either deleted or disabled by setting the **Enabled** property to false. This can be done through the portal (see <xref:ClientMaintenance>) or through the appropriate API (see <xref:identityClientCredentialClient>, <xref:identityHybridClient>, or <xref:identity-authorization-code-clients>).
```
Comment (osisoft-mbishop) [inline content/developer-guide/identity-dev/dev-guide-least-privilege.md:9]:
```suggestion
Ensure that clients that are not in use are either deleted or disabled by setting the **Enabled** property to false. This can be done through the portal (see <xref:ClientMaintenance>) or through the appropriate API (see <xref:identityClientCredentialClient>, <xref:identityHybridClient>, or <xref:identity-authorization-code-clients>).
```

Comment (osisoft-mbishop) [reply content/developer-guide/identity-dev/dev-guide-least-privilege.md:9]: small grammar thing

File: content/api-reference/identity/toc.yml
Comment (osisoft-mbishop) [inline content/api-reference/identity/toc.yml:1]: Good catch on this one.

File: content/developer-guide/identity-dev/dev-guide-role-based-access-control.md [hunk 2]
```diff
@@ -4,21 +4,23 @@ uid: accessControl
...
 
-Use an Access Control List (ACL) to manage role-based access control to entities such as namespaces and streams. ACLs control user access to entities based on their roles. Each entity has an owner, who has access for all operations regardless of the contents of the ACL. Not all entities support role-based access control.  <!--Angela Flores 6/23/21 We should not talk about unreleased functionality or future functionality in the end-user documentation. Original text "Not all entities in the AVEVA Data Hub system support role-based access control at this time, but the list will quickly grow and currently includes Namespaces and several unreleased entities." We should list all the entities that do support an ACL. How does access work for entities that don't support an ACL? -->
+Use an Access Control List (ACL) to manage role-based access control to entities such as namespaces and streams. ACLs control user access to entities based on their roles. Each entity has an owner, who has access for all operations regardless of the contents of the ACL. Not all entities support role-based access control.
+
+See [AVEVA Data Hub roles](xref:ccRoles) for details about roles.
```
Comment (osisoft-mbishop) [inline content/developer-guide/identity-dev/dev-guide-role-based-access-control.md:9]: I'd say something about these roles being default system roles:

```suggestion
See [AVEVA Data Hub roles](xref:ccRoles) for details about default system roles.
```

File: content/developer-guide/toc.yml [hunk 3]
```diff
@@ -1,5 +1,3 @@
-- name: Tenant management
-  href: tenant-dev/dev-guide-tenant-management.md
```
Comment (osisoft-mbishop) [inline content/developer-guide/toc.yml:2]: Why did you remove this topic from the TOC? Did you deem it content that isn't worthwhile (which I would agree with--I'm more curious why you removed it).

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ To add an authorization code client:
```
Comment (osisoft-mbishop) [inline content/set-up/clients/authorization-code-client-procedure.md:20]: I noticed two things about the step above:

1. "Client Type dropdown list" is vague, as the dropdown isn't actually named anything. I suggest calling it the "dropdown menu in the upper-left" or something similar.

    image

2. Rather than "Authorization Code", I think it should be "Authorization Code Clients".

File: content/set-up/clients/client-credentials-client-procedure.md [hunk 5]
```diff
@@ -20,37 +20,37 @@ To add a client-credentials client:
```
Comment (osisoft-mbishop) [inline content/set-up/clients/client-credentials-client-procedure.md:20]: Same feedback about the dropdown menu step in the previous topic--we need a locater for the dropdown menu, and the options label needs updating to "Client-Credentials Clients".

File: content/set-up/clients/client-maintenance.md [hunk 6]
```diff
@@ -18,12 +18,30 @@ To edit an existing client:
```
Comment (osisoft-mbishop) [inline content/set-up/clients/client-maintenance.md:18]: Same deal with the step above--we need to refer to the dropdown menu with a locater.

File: content/set-up/clients/client-maintenance.md [hunk 7]
```diff
@@ -18,12 +18,30 @@ To edit an existing client:
...
 
 1. Select **Save**.
 
+## Disable/enable a client
+
+To disable a client without removing it:
+
+1. In the left pane, select **Security** > **Clients**.
+
+1. In the **Client Type** dropdown list, select the appropriate client type.
```
Comment (osisoft-mbishop) [inline content/set-up/clients/client-maintenance.md:33]: Same deal.

File: content/set-up/clients/client-maintenance.md [hunk 8]
```diff
@@ -18,12 +18,30 @@ To edit an existing client:
...
 1. Select an existing client.
 
-1. Select **Edit Client**.
+1. In the `Details` pane, select **Edit Client** ![edit client](../../_icons/default/pencil.svg).
 
 1. Make any desired changes to the client configuration.
 
 1. Select **Save**.
 
+## Disable/enable a client
```
Comment (osisoft-mbishop) [inline content/set-up/clients/client-maintenance.md:27]: Nice job catching this missing task.

**Review by osisoft-mbishop — CHANGES_REQUESTED** [review]
I have few suggestions to address before merging, but your edits look good.

Your edits to the Identity chapter of the developer guide are pretty much in line with what I'm doing with SDS.

File: content/set-up/clients/client-credentials-client-procedure.md (hunk 5 above)
Comment (osisoft-mbishop) [reply content/set-up/clients/client-credentials-client-procedure.md:20]: Feel free to ignore this one based on what I said earlier.

File: content/set-up/clients/client-maintenance.md (hunk 6 above)
Comment (osisoft-mbishop) [reply content/set-up/clients/client-maintenance.md:18]: Feel free to ignore this one based on what I said earlier.

File: content/developer-guide/toc.yml (hunk 3 above)
Comment (jasnames) [reply content/developer-guide/toc.yml:2]: It's basically a duplicate of what's in the API guide, without adding anything.

File: content/api-reference/identity/toc.yml
Comment (afloresOSI) [reply content/api-reference/identity/toc.yml:1]: Can I add a comment?
